design_introspection-plugin/get_nets.cc

```diff
@@ -0,0 +1,30 @@
+#include "get_nets.h"
+
+USING_YOSYS_NAMESPACE
+
+std::string GetNets::TypeName() { return "net"; }
+
+std::string GetNets::SelectionType() { return "w"; }
+
+GetNets::SelectionObjects GetNets::ExtractSelection(RTLIL::Design* design,
+                                                    const CommandArgs& args) {
+    SelectionObjects selected_objects;
+    for (auto module : design->selected_modules()) {
+	for (auto wire : module->selected_wires()) {
```

Comment:
Getting logical nets out of Yosys is more tricky, you need to examine cell connections. This command will return wires. A net may consist of multiple wires.

Reply:
This needs more investigation, let me open a separate issue for this, so other introspection commands are not blocked by this.